spec/unit/indirector/catalog/compiler_spec.rb

```diff
@@ -506,6 +506,7 @@ def expects_no_source_metadata
             .with_parameter(:checksum, checksum_type)
             .with_parameter(:checksum_value, sha)
             .with_parameter(:source, 'puppet:///modules/mymodule/config_file.txt')
+            .with_parameter(:content_uri, 'puppet:///modules/mymodule/config_file.txt')
```

Comment:
content_uri should be of the form `'`puppet:///modules/mymodule/files/config_file.txt', perhaps an issue with how the metadata is stubbed in the test?